.github/workflows/ruby.yml

```
@@ -0,0 +1,24 @@
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Let's run builds on all PRs, regardless of their base branch.
```suggestion
on:
  push:
    branches: [ master ]
  pull_request:
```

---

Applied in https://github.com/stellar/xdrgen/pull/63/commits/2848f73ff644349cb6540796f1c801f9ff744229.